doc/devel/coding_guide.rst

```diff
@@ -47,24 +47,23 @@ Documentation
 PR Review guidelines
 ====================
 
-* If you have a commit bit, then you are trusted to use it.  Please
+* If you have commit rights, then you are trusted to use them.  Please
   help review and merge PRs!
 
 * Two developers (those with commit rights) should review all pull
-  requests.  If you are the first to review a PR please and approve of
-  the changes please edit the title to include ``'[MRG+1]'`` and use
+  requests.  If you are the first to review a PR and approve of
+  the changes, please edit the title to include ``'[MRG+1]'`` and use
   the github `'approve review'
   <https://help.github.com/articles/reviewing-changes-in-pull-requests/>`__
   tool to mark it as such.  If you are a subsequent reviewer and you
-  approve either merge (and backport if needed) or increment the
-  number in the title to ask for further review (and trigger the gh
-  'approve review').  If you do the merge please removed the
-  ``'[MRG+N']`` prefix.
+  approve, either merge (and backport if needed) or select ``'approve review'`` and 
+  increment the number in the title to ask for further review.  
+  If you do the merge, please remove the ``'[MRG+N']`` prefix.
 
 * Make sure the Travis tests are passing before merging.
 
-  - The Travis tests automatically test on all of the Python versions
-    Matplotlib supports whenever a pull request is created or updated.
+  - The Travis tests automatically test the code on all of the Python versions
```

Comment:
"Whenever a pull request is created or updated, Travis automatically runs the test suite on it for all versions of Python supported by Matplotlib."

("the code" is a bit vague)